cmd/webhook/main.go

```diff
@@ -98,6 +98,10 @@ var (
 	// policyResyncPeriod holds the interval which ClusterImagePolicies will resync
 	// This is essential for triggering a reconcile update for potentially stale KMS authorities.
 	policyResyncPeriod = flag.String("policy-resync-period", "10h", "The resync period for ClusterImagePolicies. The default is 10h.")
+
+	// trustrootResyncPeriod holds the interval which the TrustRoot will resync
+	// This is essential for triggering a reconcile update for potentially stale TUF metadata.
+	trustrootResyncPeriod = flag.String("trustroot-resync-period", "24h", "The resync period for ClusterImagePolicies. The default is 24h.")
```

Comment:
Can you use [flag.Duration](https://pkg.go.dev/flag#Duration) here?

Reply:
Yes, looks like we can. I'll update this.